recipe/meta.yaml

```
@@ -14,6 +14,8 @@ build:
 
 requirements:
   build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - {{ compiler('rust') }}
```

```suggestion
    - {{ compiler('rust') }}
    - maturin
```

You need to add maturin to build requirements.

---

Thank you!